### ISSUE TITLE
Why the training speed is decreasing gradually.

### ISSUE THREAD

**Issue by Vincento-Wang** [1231234zhan/mmlphuman#6]
<img width="1166" height="157" alt="Image" src="https://github.com/user-attachments/assets/730df922-2071-45ad-938f-2322681c8b92" />

**Comment by 1231234zhan** [comment]
Hi, after 2K iterations I start to optimize the basis and after 6K iterations LPIPS loss is used. So the speed will be slower after these two checkpoints.

what card do you use? It's normal to run 10-20 it/s for a 3090 or 4090 card. Obviously 1.04s/it is too slow. Also you could check the GPU usage to ensure that there is no big CPU bottleneck.

**Comment by Vincento-Wang** [comment]
> Hi, after 2K iterations I start to optimize the basis and after 6K iterations LPIPS loss is used. So the speed will be slower after these two checkpoints.
> 
> what card do you use? It's normal to run 10-20 it/s for a 3090 or 4090 card. Obviously 1.04s/it is too slow. Also you could check the GPU usage to ensure that there is no big CPU bottleneck.

<img width="1000" height="110" alt="Image" src="https://github.com/user-attachments/assets/7721d09f-b3b1-4b7f-bb31-5a2f39cd4a06" />  

<img width="499" height="52" alt="Image" src="https://github.com/user-attachments/assets/eeb2cd4c-ca46-4c76-bb19-8d958ce726d2" />

this is my  GPU usage and memory. is this normal ?  
I remove LPIPS and the speed is about 20 it/s all the time.

**Comment by Vincento-Wang** [comment]
I change another LIPIS pytorch model, sovled.

**Comment by chengwei-zheng** [comment]
@Vincento-Wang 

Hi, I had the same problem. Could you please share more details about your solution? Thanks!

**Comment by Vincento-Wang** [comment]
> [@Vincento-Wang](https://github.com/Vincento-Wang)
> 
> Hi, I had the same problem. Could you please share more details about your solution? Thanks!

https://github.com/richzhang/PerceptualSimilarity 
use this lipis loss solved.

**Comment by chengwei-zheng** [comment]
Thanks a lot!

**Comment by Caitianshun** [comment]
> > [@Vincento-Wang](https://github.com/Vincento-Wang)
> > Hi, I had the same problem. Could you please share more details about your solution? Thanks!
> 
> https://github.com/richzhang/PerceptualSimilarity use this lipis loss solved.



> > [@Vincento-Wang](https://github.com/Vincento-Wang)
> > Hi, I had the same problem. Could you please share more details about your solution? Thanks!
> 
> https://github.com/richzhang/PerceptualSimilarity use this lipis loss solved.

May I ask if you were able to successfully reproduce the metrics for the subject00 and avatarrex zzr datasets after changing the lipis function? After I made the change, the metrics on my side dropped significantly compared to the table in the paper.

**Comment by Caitianshun** [comment]
> Thanks a lot!

May I ask if you were able to successfully reproduce the metrics for the subject00 and avatarrex zzr datasets after changing the lipis function? After I made the change, the metrics on my side dropped significantly compared to the table in the paper.
